pages/integrations/index.tsx

```diff
@@ -487,9 +489,17 @@ export async function getServerSideProps(context) {
     },
     select: {
       id: true,
+      completedOnboarding: true,
+      createdDate: true,
     },
   });
 
+  if (
+    shouldShowOnboarding({ completedOnboarding: user.completedOnboarding, createdDate: user.createdDate })
```

Comment:
think this should work.

```suggestion
    shouldShowOnboarding(user)
```

Reply:
Used the suggestion for the types, but still only passing the required parameters instead of the entire user object.